src/main/java/com/superzanti/serversync/client/ClientWorker.java

```diff
@@ -44,13 +44,16 @@ public void run() {
 
         ServerInfo serverInfo = server.info;
 
-        switch (serverInfo.syncMode) {
+        switch (config.SYNC_MODE) {
```

Comment:
This is still using a manifest sync, the mode would still be 2.

Reply:
To check the updates without forcing the download, we set the sync mod temporarily at 3. I thought it would be easier and legit to change the configuration of the client and not the server.